en/docs/samples/sso-for-saml-apps.md

```diff
@@ -1,16 +1,82 @@
+
+![saml-sso-scenario](../assets/img/samples/saml-sso-scenario-diagram.png)
+
+Follow the steps below to deploy two sample applications and see how this works. 
+
+----
+
+## Set up Pickup Dispatch sample
+
+{!samples/pickup-dispatch-saml.md!}
+
+----
+
+## Set up Pickup Manager sample
+
+{!samples/pickup-manager-saml.md!}
+
+You are now ready to try out SAML SSO with the Pickup Dispatch and Pickup Manager sample web applications.
+
+----
+
+## Try it out
+
+1. Navigate to <http://wso2is.local:8080/saml2-web-app-pickup-dispatch.com> on your browser and click **Login**.
+
+    ![dispatch-login](../assets/img/samples/dispatch-login.png)
+
+2. You will be redirected to the login page of WSO2 Identity Server. Log in using your WSO2 Identity Server credentials (admin/admin). Provide the required consent.
+You will be redirected to the Pickup Dispatch application home page.
+
+3. Now, if you navigate to <http://wso2is.local:8080/saml2-web-app-pickup-manager.com> and click **Login**, you can see that user has been automatically logged in to this application without being prompted for user credentials.
+
+You have successfully configured SAML Single Sign-On for two web applications using WSO2 Identity Server as the identity provider. 
+
+----
+
+## Configure claims
+
+Additionally, you can also configure claims for the service providers.
+
+!!! Info
+        For more information, see
+        [Configuring Claims for a Service
+        Provider](insertlink).
+
+1. On the **Main** menu of the management console, click **Service Providers**>**List** and **Edit** the "pickup-dispatch" service provider.
+
+2. Expand the **Claim Configuration** section in the service provider form.
+
+3. You can select the claims that must be sent to the service provider. Select **Use Local Claim Dialect** and click **Add Claim URI**.
+
+4. Add the following claims as **Requested Claims**. 
+	1. http://wso2.org/claims/fullname
+	2. http://wso2.org/claims/emailaddress
+
+5. Select `http://wso2.org/claims/fullname` as the **Subject claim URI** and click **Update** to save the service provider configurations. 
+
+    ![dispatch-configure-claims](../assets/img/samples/dispatch-configure-claims.png)
+
+6. Now, logout of the **Pickup Dispatch** and **Pickup Manager** applications.
+
+7. Access <http://wso2is.local:8080/saml2-web-app-pickup-dispatch.com> on your browser and click **Login**.
+
+8. Note that the user is now prompted for consent to share the **Email Address** claim value with the sample application.  
+
+    ![dispatch-email-consent](../assets/img/samples/dispatch-email-consent.png)
+
+This means that you have successfully configured additional claims for your service provider.
```

Comment:
Shall we change this to 
Now you have successfully configured additional claims for your service provider.